### ISSUE TITLE
backup on Beset needs to report the backup file

### ISSUE THREAD

**Issue by urapadmin** [urapadmin/kiosk#657]
currently useless if one has no idea what the name of the backup file is in the end since nobody has access to the server's file system.

**Comment by urapadmin** [comment]
### kiosk v0.8.3
backup reports path and filename and offers a copy to clipboard button now. That way the filename can be written down somewhere and later it can be coped into the restore dialog.

**Comment by lbestock** [comment]
I would like to backup PVD, but the wiki instructions for backing up relate only to projects on local servers. Please instruct me what to do both to backup and to test this.

**Comment by urapadmin** [comment]
pretty much the same as with local servers. I have added a paragraph on online servers.

**Comment by urapadmin** [comment]
it has absolutely nothing to do with your mac. You just backup. The result ends up on the server. You are always dealing with the server‘s filesystem, not with your own. Not differently with local servers.

**Comment by lbestock** [comment]
![image](https://user-images.githubusercontent.com/49291909/88686657-4dc34900-d0c5-11ea-929b-1eee18f6a0e3.png)
Should I get that?

**Comment by urapadmin** [comment]
is that a rhetoric question?

**Comment by urapadmin** [comment]
"rhetorical", sorry. But mine was rhetorical, too. 
More important:
### kiosk 0.8.11
fixed.

**Comment by lbestock** [comment]
I just backed up PVD in both places. On Beset it worked as expected, with the pre-set backup location (e:\test\backup) there and no problem and the file given for me to copy.

Then I went to the local PVD server. The same file path was given there (e:\test\backup) but that is wrong, and so clicking failed. I went to the server to check where it should be, and rewrote the path as z:\backup\pvd and it backed up properly, giving me the file name. But when I tried to backup again it had reverted to the incorrect e:\test\backup path.
